onnxruntime/core/providers/xnnpack/nn/conv_base.cc

```diff
@@ -154,6 +154,26 @@ Status CreateXnnpackKernel(const ConvAttributes& conv_attrs,
         flags,
         code_cache, weights_cache,
         &p);
+  } else if (conv_type == OpComputeType::op_compute_type_fp16) {
+    const auto* B_data = Bias ? Bias->Data<MLFloat16>() : nullptr;
+    const float output_min = -65504.0;
+    const float output_max = 65504.0;
```

Comment:
Where are these values coming from? I would have expected we use something based on foutput_min/foutput_max so any clip parameters (from a fusion of two nodes) are honoured.

Reply:
It's calculated by FP16 format, 1 sign bit, 5 exponent bits and 11 mantissa bits.

Reply:
I just checked that tensorflow is using 65504 directly
https://github.com/tensorflow/tensorflow/blob/47dc9d146e99f5180906d8bd1b0c0291fa947d23/tensorflow/python/keras/layers/dense_attention.py#L126

Reply:
And it looks that we can't get the FP16 max/min value by std::numeric_limits<T> like u8s8

Reply:
updated to `const auto output_min = clip_min_max ? onnxruntime::math::floatToHalf(clip_min_max->first) : -65504.0;`

Reply:
remove onnxruntime::math::floatToHalf because xnnpack will convert float to float16